docs/mint.json

```diff
@@ -146,6 +146,12 @@
         "pages/home/api-reference/delete-oauth-app"
       ]
     },
+    {
```

Comment:
Can you delete the old passthrough APIs too?

Reply:
deleted them 👍